MCFM-JHUGen/src/User/mdata.f

```diff
@@ -511,3 +511,14 @@ block data block_epinv
       end
 ************************************************************************
 
+************************************************************************
+*     Dim. Reg. parameter epsilon, used for checking the proper        *
+*      operation of the NLO code in the program                        *
+************************************************************************
+      block data factorization_scale
+      implicit none
+      include 'facscale.f'
+      data facscale_low/10d0/
+      data facscale_high/40d0/
+      end
```

Comment:
Please separate this block into its own file to avoid unnecessarily bloating mdata.